src/pages/user-profile-page.js

```diff
@@ -0,0 +1,334 @@
+
+  static get styles() {
+    return [
+      cssStyles,
+      css`
+        * {
+          margin: 0px;
+          padding: 0px;
+        }
+        .jumbotron {
+          padding: 30px;
+          background-color: #f3f3f3;
+        }
+
+        .center {
+          text-align: center;
+        }
+
+        .jumbotron #bio {
+          font-size: 12px;
+          font-weight: light;
+
+          color: #777;
+        }
+
+        .jumbotron #name {
+          font-size: 1.5em;
+          margin: 5px;
+          font-weight: 600;
+          padding: 0;
+          color: Black;
+        }
+
+        .all-content-div {
+          padding-top: 20px;
+          display: flex;
+        }
+
+        .tag-section {
+          width: 20%;
+          font-size: 0.9em;
+        }
+
+        .tag-section div {
+          background-color: #f3f3f3;
+          padding: 12px;
+        }
+
+        .tag-section h4 {
+          font-size: 1em;
+          font-weight: normal;
+          margin: 0;
+          margin-bottom: 6px;
+        }
+
+        .content-section {
+          width: 80%;
+          margin-right: 20px;
+        }
+
+        .feed-buttons button {
+          margin: 0;
+          padding: 16px;
+          color: grey;
+          border-bottom: 2px solid #fff;
+          border: none;
+          font-size: 1em;
+          background-color: #fff;
+        }
+
+        .feed-buttons .active {
+          border-bottom: 2px solid var(--theme-color);
+          color: var(--theme-color);
+        }
+        .avatar {
+          vertical-align: middle;
+          width: 125px;
+          height: 125px;
+          border-radius: 50%;
+        }
+
+        .content-section hr {
+          margin: 0;
+          padding: 0;
+          background-color: #adadad;
+        }
+
+        user-tag {
+          padding: 0px 24px;
+        }
+
+        .pages-div {
+          margin-top: 5px;
+          display: flex;
+          flex-wrap: wrap;
+        }
+
+        .article-div {
+          position: relative;
+        }
+
+        .jumbotron-wrap {
+          position: relative;
+        }
+
+        .edit-settings {
+          position: absolute;
+          bottom: 0;
+          right: 0;
+          padding: 6px 10px;
+          border: 1px solid #777aa1;
+          color: #777aa1;
+          cursor: pointer;
+        }
+
+        .edit-settings:hover {
+          background-color: #777aa1;
+          color: #fff;
+        }
+      `
+    ];
+  }
+
+  tabsChange(e) {
+    let buttons = e.target.parentNode.children;
+    buttons[0].classList.remove("active");
+    buttons[1].classList.remove("active");
+    e.target.classList.add("active");
+
+    if (this.isToken) {
+      if (e.target.innerText === "My Articles") {
+        this.articles = this.myArticles;
+        this.pages = this.myPages;
+        this.activeTab = "my";
+      } else {
+        this.articles = this.favArticles;
+        this.pages = this.favPages;
+        this.activeTab = "fav";

```

Comment:
Lets import the hardcoded strings such as active, fav from a separate constants file